packages/suite/src/utils/suite/storage.ts

```diff
@@ -31,19 +31,7 @@ export const serializeDevice = (device: AcquiredDevice, forceRemember?: true) =>
  */
 export const serializeCoinjoinSession = (coinjoinAccount: CoinjoinAccount) => {
```

Comment:
Let's refactor this a bit:

```ts
export const serializeCoinjoinSession = (coinjoinAccount: CoinjoinAccount) => {
    const { session, transactionCandidates, ...propertiesToSave } = coinjoinAccount;
    return propertiesToSave;
};
```

If you go for this, please also adjust the comment and remove this line from `saveCoinjoinAccount`:
`delete serializedAccount.transactionCandidates; // do not store tx candidates`

Reply:
fbb5905c58538397db0bf98750fe920977060beb